libvast/native-plugins/drop.cpp

```diff
@@ -107,10 +119,10 @@ class plugin final : public virtual transform_plugin {
   // transform plugin API
   [[nodiscard]] caf::expected<std::unique_ptr<transform_step>>
   make_transform_step(const record& options) const override {
-    if (!options.contains("fields"))
+    if (!options.contains("fields") && !options.contains("schemas"))
```

Comment:
```suggestion
    if (!(options.contains("fields") || options.contains("schemas")))
```

Nit, but more idiomatic.